apps/mull-ui/src/app/app.tsx

```diff
@@ -44,6 +45,16 @@ export const App = () => {
             <EventCard event={dummyEvent} style={{ marginBottom: '1rem' }} />
           </div>
         </Route>
+        <Route
+          exact
+          path="/events/:id"
+          children={
+            <EventPage
+              eventImageURL={'https://i.ytimg.com/vi/-Mb3FoIlTCY/maxresdefault.jpg'}
```

Comment:
Add a TODO since this placeholder will need to be removed once the file download is implemented.
```suggestion
              // TODO: Remove placeholder once file download is done
              eventImageURL={'https://i.ytimg.com/vi/-Mb3FoIlTCY/maxresdefault.jpg'}
```